web/cypress/integration/apd/apd-activities/activities.spec.js

```diff
@@ -0,0 +1,236 @@
+/// <reference types="cypress" />
```

Comment:
can you remove this file from this branch and just work on it in your new branch?